torch/_inductor/output_code.py

```diff
@@ -371,8 +375,15 @@ def __init__(
         self.mutated_inputs = OrderedSet(graph.mutated_inputs)
         self.mutated_input_idxs = OrderedSet(graph.mutated_input_idxs)
         if has_frozen_params(gm):
+            # When freezing turns on, the frozen parameters can be fetched
```

Comment:
@leslie-fang-intel sorry I don't understand the problem that you're fixing here. The original intent of this code was to support freezing and caching together. The idea here is:
1) If freezing was not applied, we just store the constants with the cache entry and set them as attributes on the module when we load from cache.
2) But if there are frozen params, it would prevent caching the same graph with different params. So instead, we essentially store the names of the constants. Then on a cache hit, we look up those names in the current gm and set those attributes.

So on the path that has frozen params, I'm confused to see that we're storing both in the cache entry. Is the problem that there are some constants that are not in the `allocated_constant_name` map? If so, is that the real problem?

Reply:
Hi @masnesral, thanks for the comment. The issue we met here https://github.com/pytorch/pytorch/issues/143144 here is due to the new constants added during the lowering phase into the `GraphLowering`, for example

https://github.com/pytorch/pytorch/blob/14fe1f719026af1e6d0020c5ab1c85a5048e2eba/torch/_inductor/codegen/cpp_gemm_template.py#L927

so the name of these constant stores in `allocated_constant_name` but not in current `GraphModule`. In this PR, we only store the new added constant (not all constant) in the lowering phase into the `cache entry`.

Reply:
> But if there are frozen params, it would prevent caching the same graph with different params. 

BTW: could you give a example about this issue?

Reply:
@leslie-fang-intel Oh I see; thanks. We skip the lowering phase on a cache hit so any constants added there wouldn't be available to look up in the GM? Ok, I think we should solve this differently. I'd propose that we make the current implementation store all constants in the cache entry _except_ for the frozen params. Would you mind taking a look at https://github.com/pytorch/pytorch/pull/143808 and see if it solves our problem?